DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Configuration/PreviewCertificate.cshtml

```diff
@@ -0,0 +1,101 @@
+@{
+  Layout = null;
+}
+
+<!DOCTYPE html>
+
+<html lang="en">
+<head id="Head1">
+  <title>
+    Digital Learning Solutions Certificate
+  </title>
+  <link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/previewCertificate.css")" asp-append-version="true">
+  <link rel="stylesheet" href="@Url.Content("~/css/nhsuk.css")" asp-append-version="true">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous" />
+</head>
+<body>
+<div class="maincontentcontainer">
+  <div class="container">
+    <div class="row no-print">
+      <div class="col">
+        <button class="nhsuk-button nhsuk-u-margin-top-2 nhsuk-u-margin-bottom-3 float-right">Download PDF</button>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <div class="card card-default">
+          <div class="card-body">
+            <div class="row">
+              <div class="col">
+                <div class="p-4 float-right">
+                  @if (Model.CentreLogo != null) {
+                    <img id="Logo" src="data:image;base64,@Convert.ToBase64String(Model.CentreLogo)" alt="Centre Logo Image" />
+                  } else {
+                    <img id="Logo" src="~/images/itspCertificateLogo.png" alt="Organisation Logo" />
+                  }
+                </div>
+              </div>
+            </div>
+            <div class="text-center style1">
+              <div class="clearfix"></div>
+              <p class="style4">&nbsp;</p>
+              <p class="style4">&nbsp;</p>
+              <p class="style3">This is to certify that </p>
+            </div>
+            <div class="text-center style4">
+              <p class="style2">
+                @Model.DelegateName
+              </p>
+            </div>
+            <div class="text-center style4">
+              <p>&nbsp;</p>
+              <p class="style3">has completed</p>
+              <p class="style2">
+                @Model.CourseName
+              </p>
+              <p class="style3">
+                @Model.CertificateModifier
+              </p>
+              <p class="style2">
+              </p>
+            </div>
+            <div class="text-center ">
+              <p class="style4">&nbsp;</p>
+              <p class="style3">
+                on
+                @Model.CompletionDate.ToString("dddd, dd MMMM yyyy")
+              </p>
+              <p class="style4">&nbsp;</p>
+              <p class="style4">
+                @if (Model.SignatureImage != null) {
+                  <img src="data:image;base64,@Convert.ToBase64String(Model.SignatureImage)" alt="Centre Manager Signature Image" />
+                }
+              </p>
+              <p class="style4">&nbsp;</p>
+              <p class="style4 nhsuk-u-font-size-16">
+                @Model.CentreContactName, Digital Learning Solutions Centre Manager

```

Comment:
We happy with this reading "Not set, Digital Learning Solutions Centre Manager"?

Reply:
I'd forgotten I did that. Confirmed with Kevin we just want it blank

Reply:
When the name is null, will this now show ", Digital Learning Solutions Centre Manager" (emphasis on the comma)

Reply:
Yes, as per the old one

Reply:
Sounds a bit nutty, but fair enough.